README.md

```diff
@@ -13,37 +15,52 @@ npm install restful.js
 
 Include `restful.min.js` to the HTML, and the `restful` object is now available in the global scope:
 
+Standalone version:
```

Comment:
I don't think we should specify the two `script` tags, especially because user should modify them to adapt the Bower path.  To ease reading, I would rather specify that there are two different builds, one embedding `fetch`, which is the next generation Ajax requester.